packages/lib/delegationCredential/server.ts

```diff
@@ -211,13 +215,32 @@ export async function getAllDelegationCredentialsForUser({ user }: { user: { ema
   return delegationCredentials;
 }
 
+export async function getAllDelegationCredentialsForUser({ user }: { user: { email: string; id: number } }) {
```

Comment:
Added a fn that by default won't include `delegatedTo` which holds the serviceAccount key